iis-wf/iis-wf-ingest-pmc/src/test/java/eu/dnetlib/iis/wf/ingest/pmc/metadata/MetadataImporterMain.java

```diff
@@ -29,30 +32,52 @@ public class MetadataImporterMain {
     
     private static final String encoding = "utf-8"; 
     
+    private static final String parsedFileSuffix = ".parsed";
+    
     public static void main(String[] args) throws IOException, ParserConfigurationException, SAXException {
         if (args.length == 1) {
             if (StringUtils.isBlank(args[0])) {
                 throw new RuntimeException("No valid location provided!");
             }
             
-            ExtractedDocumentMetadata.Builder builder  = ExtractedDocumentMetadata.newBuilder();
-            builder.setId("");
-            builder.setText("");
-            extractMetadata(getFileContent(args[0]), builder);
-            
-            DatumWriter<ExtractedDocumentMetadata> writer = new SpecificDatumWriter<>(ExtractedDocumentMetadata.class);
-            ByteArrayOutputStream stream = new ByteArrayOutputStream();
-            Encoder jsonEncoder = EncoderFactory.get().jsonEncoder(ExtractedDocumentMetadata.getClassSchema(), stream);
-            writer.write(builder.build(), jsonEncoder);
-            jsonEncoder.flush();
-            System.out.println(new String(stream.toByteArray(), encoding));
+            File file = new File(args[0]);
+            if (file.isDirectory()) {
+                String[] fileLocs = file.list(new FilenameFilter() {
+                    
+                    @Override
+                    public boolean accept(File dir, String name) {
+                        return !name.endsWith(parsedFileSuffix);
+                    }
+                });
```

Comment:
The filter would be more readable as a lambda expression, I think.

Reply:
Replaced with:
```
String[] fileLocs = file.list((dir, name) -> !name.endsWith(parsedFileSuffix));
```